src/blocks/card.scss

```diff
@@ -0,0 +1,100 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+}
+
+.card {
+  padding: 16px;
+  padding-top: 32px;
```

Comment:
There is a redundancy in the padding definition for the `.card` class. You have both `padding: 16px;` and `padding-top: 32px;`. The `padding-top` will override the top padding value from the `padding` shorthand. Consider removing the redundant `padding-top` or adjusting the `padding` shorthand to achieve the desired effect.